com.rawtherapee.RawTherapee.json

```diff
@@ -0,0 +1,268 @@
+{
+  "app-id": "com.rawtherapee.RawTherapee",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+  "sdk": "org.gnome.Sdk",
+  "command": "rawtherapee",
+  "rename-desktop-file": "rawtherapee.desktop",
+  "rename-icon": "rawtherapee",
+  "finish-args": [
+    /* X11 + XShm access */
+    "--share=ipc",
+    "--socket=x11",
+    /* Wayland */
+    "--socket=wayland",
+    /* Note playback */
+    "--socket=pulseaudio",
```

Comment:
Audio playback in raw photo processing program?

Reply:
uh lol. I thought i had cleaned that section up.